superset/utils/core.py

```diff
@@ -174,8 +174,16 @@ class GenericDataType(IntEnum):
     # ROW = 7
 
 
+class DatasourceType(Enum):
```

Comment:
you're going to want to inherit from (str, Enum) so that you can show the string representation later.